ManageIQ/manageiq
# [EUWE] add storage role to product features

In accordance with Bugzilla ticket https://bugzilla.redhat.com/show_bug.cgi?id=1395528
the Storage entries are missing from the list of Product Features shown
when attempting to add a new role.

This simple fix restores the Storage entries to the list of Product Features.

# **Links**

https://bugzilla.redhat.com/show_bug.cgi?id=1395528

** Steps for Testing/QA**

Click on "Configuration" in the menu in the upper right-hand side of the screen.
Click on "Access Control" in the accordion menu on the left, then click on "Roles".
Configuration -> Add a New Role in the left-hand side drop-down menu.
"Storage" should be visible under "Product Features (Editing)", as well as sub-items under Storage.

@chessbyte cherry-picked and resolved conflicts.
